packages/components/organisms/f-checkout/src/components/Address.vue

```diff
@@ -135,11 +163,19 @@ export default {
             return this.isFieldEmpty(VALIDATIONS.address, 'postcode');
         },
 
+        isAddressAdministrativeAreaEmpty () {
+            return this.isFieldEmpty(VALIDATIONS.address, 'administrativeArea');
+        },
+
         /*
         * Checks that postcode is a valid postcode and that the field is not empty.
         */
         isAddressPostcodeValid () {
             return (!this.$v[VALIDATIONS.address].postcode.$dirty || this.$v[VALIDATIONS.address].postcode.isValidPostcode) && !this.isAddressPostcodeEmpty;
+        },
+
+        shouldShowAdministrativeArea () {
+            return this.tenant === 'au';
```

Comment:
Should this include NZ

Reply:
When I saw `State` mentioned below realised what that is! Assuming comment irrelevant, resolving.